src/strip_output.py

```diff
@@ -0,0 +1,25 @@
+import os
+from pathlib import Path
+import subprocess
+
+path = Path("..").cwd() / "src"
```

Comment:
Cleaner if `strip_output` lives in contrib and takes input and output path as argument (with `notebook/src` and `notebooks/build` be provided via Makefile when the user types `make scrub`).